src/scf-release/src/acceptance-tests-brain/test-scripts/017_syslog_forwarding_test.rb

```diff
@@ -115,7 +115,7 @@ def emit_log_entries
     # (B) Configure and run the log receiver pod
 
     pod_info = JSON.load capture("kubectl get pod -n #{$KUBERNETES_NAMESPACE} #{ENV['HOSTNAME']} -o json")
-    image = pod_info['spec']['containers'].find { |container| container['image'] }['image']
+    image = 'opensuse/leap'
```

Comment:
I don't think we need `pod_info` anymore

Reply:
Heh. Will change.

Reply:
Done with commit a3940f56bfe3b8bb9f6f559fa972cd8c978738ea.
Also rebased to get latest minibroker things.